samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -234,11 +238,24 @@ static const char link_rsrp_usage_str[] =
 	"  -u, --unsubscribe,  Unsubscribe for RSRP info\n";
 
 static const char link_ncellmeas_usage_str[] =
-	"Usage: link ncellmeas --single | --continuous |--cancel\n"
+	"Usage: link ncellmeas --single | --continuous |--cancel [<search_type>]\n"
 	"Options:\n"
-	"      --single,     Start a neighbor cell measurement and report result\n"
-	"      --continuous, Start continuous neighbor cell measurement mode and report result\n"
-	"      --cancel,     Cancel/Stop neighbor cell measurement if still ongoing\n";
+	"      --single,       Start a neighbor cell measurement and report result\n"
+	"      --continuous,   Start continuous neighbor cell measurement mode and report result\n"
+	"      --cancel,       Cancel/Stop neighbor cell measurement if still ongoing\n"
+	"Optional options:\n"
```

Comment:
I guess an "option" is optional without being an "optional option". :) Also these are valid for single and continuous measurements, but not for cancel.
```suggestion
	"Search type options, can be used when starting measurements:\n"
```

Reply:
Changed.